src/Tickets/Commerce/Gateways/PayPal/Hooks.php

```diff
@@ -126,46 +123,6 @@ public function include_success_page_vars( $template_vars, $shortcode ) {
 		return $template_vars;
 	}
 
-	/**
-	 * Include the Client JS SDK script into checkout.
-	 *
-	 * @since 5.1.9
-	 *
-	 * @param string           $file     Which file we are loading.
-	 * @param string           $name     Name of file file
-	 * @param \Tribe__Template $template Which Template object is being used.
-	 *
-	 */
-	public function include_client_js_sdk_script( $file, $name, $template ) {
-		echo tribe( Buttons::class )->get_checkout_script();
-	}
-
-	/**
-	 * Include the payment buttons from PayPal into the Checkout page.
-	 *
-	 * @since 5.1.9
-	 *
-	 * @param string           $file     Which file we are loading.
-	 * @param string           $name     Name of file file
-	 * @param \Tribe__Template $template Which Template object is being used.
-	 */
-	public function include_payment_buttons( $file, $name, $template ) {
-		$this->container->make( Buttons::class )->include_payment_buttons( $file, $name, $template );
-	}
-
-	/**
-	 * Include the advanced payment fields from PayPal in the Checkout page.
-	 *
-	 * @since 5.2.0
-	 *
-	 * @param string           $file     Which file we are loading.
-	 * @param string           $name     Name of file file
-	 * @param \Tribe__Template $template Which Template object is being used.
-	 */
-	public function include_advanced_payments( $file, $name, $template ) {
-		$this->container->make( Buttons::class )->include_advanced_payments( $file, $name, $template );
-	}
```

Comment:
Why was this removed? Why are we touching PayPal codebase?

Reply:
Same comment from above.